packages/react-output-target/src/react/ssr.tsx

```diff
@@ -55,52 +48,49 @@ export const createComponentForServerSideRendering = <I extends HTMLElement, E e
       stringProps += ` ${key}="${value}"`;
     }
 
-    const toSerialize = `<${options.tagName}${stringProps}></${options.tagName}>`;
+    let serializedChildren = '';
+    const toSerialize = `<${options.tagName}${stringProps} suppressHydrationWarning="true">`;
+    try {
+      const awaitedChildren = await resolveComponentTypes(children);
+      serializedChildren = ReactDOMServer.renderToString(awaitedChildren);
+    } catch (err: unknown) {
+      const error = err instanceof Error ? err : new Error('Unknown error');
+      console.log(
```

Comment:
~ No action needed. Is a serialization failure actionable for the developer? If so, thoughts on elevating this to a warning?

Reply:
Good suggestion, I can tweak this with my next PR.